hudi-common/src/main/java/org/apache/hudi/common/model/HoodieFileFormat.java

```diff
@@ -36,4 +49,9 @@
   public String getFileExtension() {
     return extension;
   }
+
+  public static boolean isBaseFile(Path path) {
```

Comment:
do you think we should move this to FSUtils?

Reply:
I feel its closer to file format but am open to move it if you feel otherwise. Also, the isLogFile    check is in FSUtils since it uses some regex instead of the file extension.

Reply:
yeah, thats what I thought too. 
isLogFile, getFileExtensionFromLog are in FSUtils.

Reply:
moved to FSUtils.